src/data/tests/EnterpriseAppTestData/context.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable max-len */
+/* eslint-disable react/jsx-filename-extension */
```

Comment:
We shouldn't ignore this rule. If this file is rendering JSX, it should be using the `.jsx` file extension.